rumps/rumps.py

```diff
@@ -657,6 +666,47 @@ def key(self):
         return self._menuitem.keyEquivalent()
 
 
+class SliderMenuItem(MenuItem):
+
+    def __new__(cls, *args, **kwargs):
+        if args and isinstance(args[0], SliderMenuItem):  # can safely wrap SliderMenuItem instances
+            return args[0]
+        return super(SliderMenuItem, cls).__new__(cls, *args, **kwargs)
+
+    def __init__(self, title, value=50, minValue=0, maxValue=100, callback=None, name=None, dimensions=(180, 15)):
+        if isinstance(title, SliderMenuItem):  # don't initialize already existing instances
+            return
+        self._slider = NSSlider.alloc().init()
+        self._slider.setMinValue_(minValue)
```

Comment:
Can we change `minValue` and `maxValue` to follow a pythonic structure? `min_value` and `max_value`?